components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -457,7 +457,7 @@ void MAMMicrophysics::initialize_impl(const RunType run_type) {
     auto linoz_cariolle_pscs =
         buffer_.scratch[7];  // Cariolle parameter for PSC loss of ozone [1/s]
 
-    auto ts = timestamp();
+    auto ts = start_of_step_ts();
```

Comment:
@singhbalwinder I left all use of timestamp _mostly_ unchanged in mam. Meaning that I am using `start_of_step_ts` (rather than `end_of_step_ts`), but since timestamps are updated _within subcycled processes_ this is not 100% equivalent to what was done before. Hence, the non-bfbness.

But you may want to investigate whether we should switch all of these to `end_of_step_ts`.

Reply:
I just noticed this fcn is inside `initialize_impl`, where start/end of timestep have same timestamp.